sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/GalleriesOperations.cs

```diff
@@ -134,7 +138,7 @@ public async Task<AzureOperationResponse<Gallery>> UpdateWithHttpMessagesAsync(s
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<Gallery>> GetWithHttpMessagesAsync(string resourceGroupName, string galleryName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<Gallery>> GetWithHttpMessagesAsync(string resourceGroupName, string galleryName, string select = default(string), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This is a breaking change - we should add an overload in customization that matches the old parameters

Reply:
How is this a breaking change? Isn't this new added parameter optional?

Reply:
Nvm comment above ^ I can see why it is a breaking change now. I've added some updates for this, please take a look.